src/misosoupy/setup_misosoupy.py

```diff
@@ -76,3 +78,32 @@ def get_path_to_assets():
         return Path(resources.files(misosoupy) / "assets")
     else:
         return Path(pkg_resources.resource_filename("misosoupy", "assets"))
+
+# Read in config file
+# (code from https://medium.com/@lelambonzo/simplifying-configuration-file-parsing-in-python-ef8e2144b3b3)
+def parse_config_file(config_path): 
+    config = configparser.ConfigParser()
+    config.read(config_path)
+
+    steps_to_complete = {}
+    for key, value in config['STEPS'].items():
+        if value.lower() in 'true':
+            steps_to_complete[key] = True
+        elif value.lower() in 'false':
+            steps_to_complete[key] = False
+
+    screen_parameters = {}
+    for key, value in config['SCREEN'].items():
+        if value.isdigit():
+            screen_parameters[key] = int(value)
+        elif value.lower() in 'true':
+            screen_parameters[key] = True
+        elif value.lower() in 'false':
+            screen_parameters[key] = False
+        else:    
+            try:
+                screen_parameters[key] = float(value) 
+            except:
```

Comment:
Usually bare excepts are discouraged, I would try to at least specify the error (and maybe there is a way to avoid a try and except completely): https://python101.pythonlibrary.org/chapter7_exception_handling.html#:~:text=In%20Python%2C%20this%20is%20known,a%20division%20by%20zero%20error.

Reply:
helpful, thanks!